chapters/unitexpressions.tex

```diff
@@ -94,7 +94,45 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 \section{Unit Checking}\label{unit-checking}
 
 How to verify that units are used in compatible ways is current not fully determined by the specification.
-This section gives rules for certain situations, but in general tools are expected to reason according to standard dimensional analysis (from here on referred to as just \firstuse{dimensional analysis}).
+This section gives rules for certain situations, but in general tools are expected to reason according to standard dimensional analysis..
+
+
+\subsection{Standard Dimensional Analysis}\label{standard-dimensional-analysis}
+
+This section gives an incomplete characterization of \emph{standard dimensional analysis}, also referred to as just \firstuse{dimensional analysis}.
+What is described below applies to unit checking in Modelica -- \emph{standard dimensional analysis} could have additional meanings in other contexts.
+While Modelica has a concept of \willintroduce{empty units} (described in later sections), standard dimensional analysis only deals with non-empty units such as \lstinline!"m"!, \lstinline!"m/s"!, or \lstinline!"1"!.
+It consists of two parts:
+\begin{itemize}
+\item
+  Unit compatibility requirements.
+\item
+  Rules for deriving the unit of an expression.
+\end{itemize}
+
+Examples of unit compatibility requirements that must be met in Modelica:
+\begin{itemize}
+\item
+  The expression of a declaration equation must have the same unit as the component to which it belongs.
+\item
+  The two sides of an equation or assignment statement must have the same unit.
```

Comment:
This is called *dimensional homogeneity* (see https://en.wikipedia.org/wiki/Dimensional_analysis)

Reply:
I've reformulated so that the terminology is used.  OK?

Reply:
Yes OK (I left a thumbs up, that probably was unclear)

Reply:
OK.  Resolving.